packages/components/src/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,112 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import * as React from "react";
+import Button from "../Button";
+import Tooltip from "./Tooltip";
+
+const defaultArgs = {
+  content: (
+    <span>
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit.{" "}
+      <b>Donec a erat eu augue consequat eleifend non vel sem.</b> Praesent
+      efficitur mauris ac leo semper accumsan.
+    </span>
+  ),
+  children: <Button className="mx-32 my-32">Tooltip trigger</Button>,
+  placement: "right",
+  visible: true,
+};
+
+export default {
+  title: "Tooltip",
+  component: Tooltip,
+  args: defaultArgs,
+  argTypes: {
+    variant: {
+      control: {
+        type: "select",
+        options: ["light", "dark"],
+      },
+    },
+    placement: {
+      control: {
+        type: "radio",
+        options: ["left", "right", "top", "bottom"],
+      },
+    },
+  },
```

Comment:
I think you don't need these in this repo. Try taking them out and see if you still see the controls on the first story on the `docs` tab.

Reply:
Ohh you are right, removed